src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -287,6 +187,73 @@
       },
       "required": [ "value", "algorithm" ]
     },
+    "invocation": {
+      "description": "The runtime environment of the analysis tool run.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "parameters": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
+          "type": "string"
+        },
+
+        "startTime": {
+          "description": "The date and time at which the run started. See \"Date/time properties\" in the SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "endTime": {
+          "description": "The date and time at which the run ended. See \"Date/time properties\" in the  SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "correlationId": {
+          "description": "An identifier that allows the run to be correlated with other artifacts produced by a larger automation process.",
+          "type": "string"
+        },
+
+        "machine": {
+          "description": "The machine that hosted the analysis tool run.",
+          "type": "string"
+        },
+
+        "processId": {
+          "description": "The process identifier for the analysis tool run.",
+          "type": "integer"
+        },
+
+        "workingDirectory": {
+          "description": "The working directory for the analysis rool run.",
+          "type": "string",
+          "default": { }
+        },
+
+        "environmentVariables": {
+          "description": "The environment variables associated with the analysis tool process, expressed as key/value pairs.",
+          "type": "object",
+          "default": { }
+        },
+
+        "properties": {
+          "description": "Key/value pairs that provide additional information about the run.",
+          "type": "object",
+          "default": { }
+        },
+
+        "tags": {
+          "description": "A set of distinct strings that provide additional information about the run.",
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "type": "string",
+            "default": [ ]
```

Comment:
this "default" belongs on "tags", not under tags.items. It is the default value for the tags property, not the default value for each of its items. (The items are strings, not arrays.)